docs/introduction.md

```diff
@@ -0,0 +1,18 @@
+Studio is a tool to develop AsyncAPI documentation. It allows you to visualize the flow, events, publishers, subscribers as you design your API.
+
+# Studio Use Cases
+
+- Create and visualize event-driven AsyncAPI on the go.
+- Realtime visualization and documentation for your APIs.
+
+
+# Studio Advantages 
+- User-friendly.
+- Straightforward and effortless in building APIs.
+- Interactive AsyncAPI documentation.
+
+# Process
+
+To start using Studio, [create an AsyncAPI file](https://www.asyncapi.com/docs/tutorials/streetlights#creating-the-asyncapi-file) for your API.
```

Comment:
```suggestion
To start using Studio, [create an AsyncAPI file](https://www.asyncapi.com/docs/tutorials/create-asyncapi-document#create-AsyncAPI-document) for your API. 
```

This link won't work when we merge the new 4 GSoD tutorials to replace the Streetlights tutorial: https://github.com/asyncapi/website/pull/1023/files